libs/application/templates/reference-template/src/forms/ExampleForm.ts

```diff
@@ -114,11 +117,24 @@ export const ExampleForm: Form = buildForm({
       id: 'confirmation',
       name: 'Staðfesta',
       children: [
-        buildIntroductionField({
-          id: 'overview',
-          name: 'Takk fyrir að sækja um',
-          introduction:
-            'Með því að smella á "Senda" hér að neðan, þá sendist umsóknin inn til úrvinnslu. Við látum þig vita þegar hún er samþykkt eða henni er hafnað.',
+        buildMultiField({
+          name: '',
+          children: [
+            buildSubmitField({
+              id: 'submit',
+              placement: 'footer',
+              name: 'Senda inn umsókn',
+              actions: [
+                { event: 'SUBMIT', name: 'Senda inn umsókn', type: 'primary' },
+              ],
+            }),
+            buildIntroductionField({
+              id: 'overview',
+              name: 'Takk fyrir að sækja um',
+              introduction:
+                'Með því að smella á "Senda" hér að neðan, þá sendist umsóknin inn til úrvinnslu. Við látum þig vita þegar hún er samþykkt eða henni er hafnað.',
+            }),
+          ],
```

Comment:
@fjandakornid great spot in the other PR, I fixed it here.

Now you need to include the submit field somewhere, otherwise the application will never really be submitted...We can discuss the pros and cons here.

Reply:
Looks good to me :+1: Will do.